model/src/main/kotlin/OutputFormat.kt

```diff
@@ -52,6 +53,6 @@ fun File.mapper() =
         )
 
 /**
- * Use the mapper returned from [File.mapper] to read the [valueType] object from this file using Jackson.
+ * Use the Jackson mapper returned from [File.mapper] to read an object of type [T] from this file.
  */
-fun <T> File.readValue(valueType: Class<T>): T = mapper().readValue(this, valueType)
+inline fun <reified T : Any> File.readValue(): T = mapper().readValue(this)
```

Comment:
Nit: Space before `:`.

Reply:
That's in line with the Kotlin coding conventions (space before colon if it separates type and supertype):
https://kotlinlang.org/docs/reference/coding-conventions.html#colon

Reply:
Wow, surprising, good to know!